### ISSUE TITLE
Add the new 67-373 team as contributors

### ISSUE THREAD

**Issue by purelogiq** [sc0v/binder-app#223]
Hi @ChaseBro, @merichar, @pkoenig10,

There is a new 67-373 IS team for Spring 2017 that will be working on the project. They asked me to have them added onto this GitRepo so that they can review each other's PRs and eventually merge things to master. I met with them and they seem like nice people, they are: @eleanorwilson, @AkshayGoradia, and @blinblinblin.

I do not have permissions to add these students to this project as contributors, but, if someone else could that would be awesome. I told them to use PRs for all changes and will try to review most of them. But, again they want to be added so that they have autonomy in how they work within their group.

Thank you

**Comment by ChaseBro** [comment]
I don't have permissions 😢

**Comment by ChaseBro** [comment]
But regardless, I would say they should fork into their own repo (even if it's under the sc0v org) so that they can play and we can do merges at milestones or all at once without worrying about things being in a broken state.

**Comment by pkoenig10** [comment]
I invited them, although it looks like @chairscc already did.  Good idea creating an SCC account. Who is managing Binder this year?

And @ChaseBro not sure why you weren't an owner, changed that back.

And they should fork or work on separate branches.  Last year I found that just working on separate branches in this repo made it much easier to check out different people's changes for testing, verification, etc. than having forks.
